homeassistant/components/recorder/__init__.py

```diff
@@ -133,18 +141,24 @@ def run_information(hass, point_in_time: Optional[datetime] = None):
     conf = config[DOMAIN]
     keep_days = conf.get(CONF_PURGE_KEEP_DAYS)
     purge_interval = conf.get(CONF_PURGE_INTERVAL)
+    db_max_retries = conf[CONF_DB_MAX_RETRIES]
+    db_retry_wait = conf[CONF_DB_RETRY_WAIT]
 
     db_url = conf.get(CONF_DB_URL, None)
     if not db_url:
         db_url = DEFAULT_URL.format(hass_config_path=hass.config.path(DEFAULT_DB_FILE))
 
+
+
```

Comment:
Please remove these empty lines too.

Reply:
They have now been removed in 8112a1eadef2a658b2cca1f0313c07b22b199542 . Sorry. I'm having a hard time testing the code locally.